client/my-sites/themes/style.scss

```diff
@@ -196,6 +196,32 @@
 	animation: loading-fade 1.6s ease-in-out infinite;
 }
 
+.themes__sheet-features ul.theme-features-list {
+	text-align: center;
+	margin: -0.78em 0;
+	padding: 0;
+
+	li {
+		list-style: none;
+		display: inline-block;
+		background: #E9EFF3;
+		color: $gray-dark;
+		font-size: 0.925em;
+		margin: 0.38em 0.37rem;
+		padding: 0.25em 0.7em;
+		border-radius: 3px;
+
+		span {
+			display: block;
+
+			&:hover {
```

Comment:
This part not necessary. Would only have been necessary if we were using `<a>` tags already. My fault for confusing you :)